resnet/cifar_input.py

```diff
@@ -101,7 +109,7 @@ def build_input(dataset, data_path, batch_size, mode):
   labels = tf.reshape(labels, [batch_size, 1])
   indices = tf.reshape(tf.range(0, batch_size, 1), [batch_size, 1])
   labels = tf.sparse_to_dense(
-      tf.concat(1, [indices, labels]),
```

Comment:
OK, I'll change it to concat_v2, although that one is also getting deprecated:

https://github.com/tensorflow/tensorflow/releases/tag/v1.0.0-alpha
Deprecated tf.concat operator. Please switch to use tf.concat_v2 for now.
In the Beta release, we will update tf.concat to match argument order of
tf.concat_v2.